### PR TITLE
Test: Add FeedbackSessionsLogicTest::testIsResponseVisibleForStudentInTeam (Resolve #67)

### LINKED ISSUE #67: Test: Add FeedbackSessionsLogicTest::testIsResponseVisibleForStudentInTeam
Exercise code inside if statement (studentsEmailInTeam != null && isStudent(role))
So to speak check if student is in the list returns true if in student is in email list.

### PR DESCRIPTION
**Pull request by afTrolle**
This Resolves #67   
And I dependant on pull Request #68

### CODE REVIEW

File: src/main/java/teammates/logic/core/CoverageCounter.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package teammates.logic.core;
+
+/**
+ * A simple class that can be used to calculate coverage
+ */
+public class CoverageCounter {
+    static private int size = 0;
```
Comment (gingemarsson) [inline src/main/java/teammates/logic/core/CoverageCounter.java]: This file should not be included

File: src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java [hunk 2]
```diff
@@ -2166,4 +2157,30 @@ private void testDeleteFeedbackSessionsForCourse() {
         assertTrue(fsLogic.getFeedbackSessionsForCourse("idOfTypicalCourse1").isEmpty());
     }
 
+
+    @Test
+    public void testIsResponseVisibleForStudentInTeam() {
+        //contract
+        //Exercise code inside if statement (studentsEmailInTeam != null && isStudent(role))
```
Comment (gingemarsson) [inline src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java]: The contract should not be formulated as a description of the test case. It should be a description of the correct behavior

File: src/main/java/teammates/logic/core/CoverageCounter.java (hunk 1 above)
Comment (afTrolle) [reply src/main/java/teammates/logic/core/CoverageCounter.java]: removed

File: src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java (hunk 2 above)
Comment (afTrolle) [reply src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java]: fixed

**Pull request merged**